Gruntfile.coffee

```diff
@@ -117,4 +107,4 @@ This file is generated by `grunt build`, do not edit it by hand.
     json2 = pkg.jqueryJSON
     json1[key] = json2[key] for key of json2
     json1.author.name = pkg.author
-    grunt.file.write('chosen.jquery.json', JSON.stringify(json1, null, 2))
+    grunt.file.write('chosen.jquery.json', JSON.stringify(json1, null, 2) + "\n")
```

Comment:
We could drop this json file entirely given that the jQuery plugin registry is abandonned: http://plugins.jquery.com/